internal/search/search.go

```diff
@@ -40,3 +41,14 @@ func ClusterExists(c *opsmngr.AutomationConfig, name string) bool {
 
 	return rsFound || shardedFound
 }
+
+// IsClusterFound returns true if a cluster exists for the given name
+func IsClusterFound(clusters []atlas.Cluster, name string) bool {
```

Comment:
consider the package part of your name, `search.IsClusterFound` is probably not the best name, some alternatives
`search.AtlasClusterByName`, `search.AtlasClusterExists`